### PR TITLE
Bug 1209961 - Rename fonts to bidi-proof names.

### PR DESCRIPTION
**Pull request by autra**
This commit removes any reference to end/beginning semantics on directional
icons: left is the beginning and right is the end only for LTR languages. RTL
languages follow the exact opposite semantic. Instead, we keep the left/right in
names for these icons, and let the user of these fonts deals with bidi csss
rules accordingly.

### CODE REVIEW

**Comment by wilsonpage** [comment]
We'll need a followup in Gaia and gaia-components to do renaming.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "gaia-icons",
-  "version": "0.9.2",
+  "version": "0.10.0",
```
Comment (wilsonpage) [inline package.json]: We'll stamp version after this patch is landed, so let's revert this.

**Comment by autra** [comment]
Yep, working on that now. Btw, do you have a list of external gaia-components that uses gaia-fonts?  That will ease things :-)

**Comment by wilsonpage** [comment]
gaia-components/gaia-header is the most important one as it's used everywhere. But take a look at the [gaia-components demo page](http://gaia-components.github.io/gaia-components/).

**Pull request merged**